content/md/permissions-settings/manage-the-interface-and-actions-accesses.md

```diff
@@ -202,6 +202,18 @@ Only give UI permissions to user roles that are related to real users, not to [c
 | List Currency     | User can see all (active and inactive) currencies listed under `Settings`/`Currencies`            |
 | Toggle currencies | User can activate/deactivate currencies using the toggle button on the right side of the grid |
 
+## Permissions on measurements
+
+| Permission name | Effect on the interface |
+|---------------------------|--------------|
+| Create a measurement family    | User can create a new measurement family with the `Create` button in the `Settings`/`Measurements` |
+| Delete a measurement family with its units| User can delete a measurement family with its units with the `Delete measurement family` button in the `...` dropdown on the top of the measurement family form|
```

Comment:
```suggestion
| Delete a measurement family with its units| User can delete a measurement family with its units using the `Delete measurement family` button visible under the `...` dropdown at the top of the measurement family form|
```
at instead of on??